hardware/tv-hat/README.md

```diff
@@ -0,0 +1,58 @@
+## Getting started with the Raspberry Pi TV HAT
+
+The TV HAT has an on-board DVB-T2 tuner that allows you to receive and decode digital television streams on your Raspberry Pi. Then you can watch these streams on any computer connected to the same network as the Pi.
```

Comment:
Maybe this opening sentence should clarify that the streams can be watched on the Raspberry Pi itself _or_ on any computer connected to the same network as the Pi?